### ISSUE TITLE
invisible players are still invisible

### ISSUE THREAD

**Issue by 182exe** [182exe/clairvoyance#2]
probably impossible, comment if you know anything

**Comment by Darkar25** [comment]
Invisible player's model is not rendered at all, the shader(and the client as a whole if the invisibility plugin is advanced enough) has no way of overriding that. The only way you could detect invisible players is either through particles or ingame actions like opening chests.
TLDR; It's impossible.

**Comment by 182exe** [comment]
fuckass shader system

**Comment by Darkar25** [comment]
I agree, this is quite unfortunate. You could check out my Silverfish repo, I have conducted a pretty extensive research of what is possible with shaders. Perhaps you could find features that you would like to replicate in your own shader, or if you know some tricks or hidden features that could extend the shader capabilities you could share them.
